### DISCUSSION TITLE
Commits are not showing up

### DISCUSSION THREAD

**Discussion by Timed-AutoMaton** [community/community#81769] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

Good day, and Merry Christmas!

My commits are not showing up on my contribution timeline. Please help!

Thanks!

**Comment by leeringrose** [comment]
@Zubair-Underwood Merry Christmas!
Could you please be more specific? Maybe the screenshots would be very helpful for others to understand your problem.

**Comment by Timed-AutoMaton** [comment]
![2023-12-25_06-32](https://github.com/community/community/assets/97603324/a0105598-3c99-4e6e-b123-2477c994ba67)
![2023-12-25_06-31](https://github.com/community/community/assets/97603324/21f8fe3a-fb29-4306-888c-a9f871d4128b)
![2023-12-25_06-30](https://github.com/community/community/assets/97603324/633b62ae-89bc-4d25-ab89-c09e7a42450f)

Thank you for replying. I mean, none of my contributions are being shown up on the Contributions Timeline/graph. The last contribution that appears on my contributions timeline/graph is from December 15th. Please see the attached.

**Reply by leeringrose** [reply, marked as answer]
![292696649-21f8fe3a-fb29-4306-888c-a9f871d4128b](https://github.com/community/community/assets/132311826/7e31c6bc-aa9d-4b32-ab2d-566870587f75)
![292696645-a0105598-3c99-4e6e-b123-2477c994ba67](https://github.com/community/community/assets/132311826/06f78e0a-39b7-4a0a-8e09-0eef9d419957)

That contribution times do not display your Github name. That says `Your Name` contributed 9 mins ago, and such days ago, but not `Zubair-Underwood` contributed.
I think you have contributed recently with not your `Zubair-Underwood` Github account. I recommend you to check if your VS code app or Github Desktop (Git management client app) has logged in with your `Zubair-Underwood` account.

**Reply by Timed-AutoMaton** [reply]
@leeringrose Please see the attached, I have logged in as Zubair-Underwood. And once again the contribution that was showing up yesterday is vanished again. I have been quite active with the commits for the last two weeks.
![2023-12-25_21-58](https://github.com/community/community/assets/97603324/dba0adcf-6455-4dfb-9d85-a94722ad78e1)
![2023-12-25_21-55](https://github.com/community/community/assets/97603324/762f1ec7-67c5-43e5-a133-011e6bb2b70b)

**Reply by Timed-AutoMaton** [reply]
@leeringrose You were right. My email wasn't synced. Just fix it and it is working now. Thanks alot!

**Comment by brf153** [comment]
@Zubair-Underwood it takes some time to show the commit in the contributions chart in your GitHub profile(that green-colored timeline). Also, your commit is showing in the GitHub repo. See the commit message "learning" next to the quiz-app folder which was made 9 min ago. Your commit message will only be displayed in the folders where you have made the changes. By the way Merry Christmas everyone!

**Reply by leeringrose** [reply]
Yeah, Those're also points. However, any contribution to any specific repository or folder should be displayed on chart. Some time delayed to be shown in the chart after contribution sounds reasonable. Merry Christmas to you too @brf153 !

**Comment by Timed-AutoMaton** [comment]
@brf153 @leeringrose Thank you both for your help. You're right @brf153 that the commits I made yesterday are now appearing on my contribution timeline. But, the commits that I made since December 15th are still not appearing on my timeline.

**Reply by brf153** [reply]
@Zubair-Underwood Please do me a favor. I actually want the Galaxy Brain badge in my profile. Please mark my answer as correct, I will get a badge for it :).
